modules/policy-change-management.adoc

```diff
@@ -22,6 +22,22 @@ Change history is captured in the *Cluster History* section in the OpenShift Clu
 * Scaling the cluster persistent storage
 * Upgrading the cluster
 
+You can implement a maintenance exclusion by avoiding changes in {cluster-manager} for the following components:
+
+* Deleting a cluster
+* Adding, modifying or removing identity providers
+* Adding, modifying or removing a user from an elevated group
+* Installing or removing add-ons
+* Modifying cluster networking configurations
+* Adding, modifying, or removing machine pools
+* Enabling or disabling user workload monitoring
+* Initiating an upgrade
+
```

Comment:
How about cluster history and support?

Reply:
Cluster history is read-only by customers, and has no impact of cluster actions if something changes. Same with support, there are no impact to clusters by modifying anything on the support tab.

Cluster history and support aren't tabs/actions that need to be considered for a maintenance exclusion.